tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
     "lib": [
       "es2015",
-      "es2017"
+      "es2017",
+      "DOM"
```

Comment:
NodeJS server application can't include DOM. That's a browser only library. It implements things like `window` and `document` which we surely don't have available. I'm guessing you've enabled it here because of `atob` and `btoa` deprecation, correct? We can't use those functions. You'd need to either avoid using them completely (which I think we can) or use `Buffer.from(...).toString(...)`.

Reply:
Thanks for the information.
yes i had enabled for atob and btoa :sweat_smile: 
will make the changes.